### ISSUE TITLE
Scheduling doses to certain weekdays

### ISSUE THREAD

**Issue by MojoDodo** [CorruptedArk/did-i-take-my-meds#50]
Currently the app allows to set the schedule for a medication every X days/weeks/...
Currently meds that should be taken for instance "three times a week with a minimum time difference of 48 hours between each dose" cannot properly be scheduled with the app. 

Being able to define certain weekdays would be an easy opportunity to solve this problem.

Thanks already!

**Comment by CorruptedArk** [comment]
Wouldn't this schedule work?

You can set a weekday by choosing a start day and set the repeat period to a week

![](https://i.imgur.com/Sq4hIE6.jpg)

**Comment by CorruptedArk** [comment]
Sorry the last dose schedule is wrong, but if it was a week, it would work

**Comment by MojoDodo** [comment]
You mean adding three medication schedules with each having a repetition period of one week? 
Well, you are right, this would indeed be a proper workaround and might do the job, although in practice it might not be the smoothest option for scenario as described above ("three times a week with a minimum time difference of 48 hours between each dose" ).
In the latter case the weekly schedule may vary between Mo, Wed, Fri or Mo, Th, Sa ... But this would require to set the schedule more complex anyway ... so, thanks for the suggestion with the workaround, it's so obvious that it's easy to oversee

**Comment by CorruptedArk** [comment]
I agree, it's not smooth for your particular case, but the handling for schedules is complicated to try and cover as many options as possible, some of them will be edge cases that are difficult to to implement.

Unfortunately, making your case easy would take a lot of restructuring and likely break things for other people

**Comment by CorruptedArk** [comment]
The birth control schedules for example are a special case to make them more convenient, but they still create the schedule the same way internally
